FE/src/utils/movieMethods.ts

```diff
@@ -0,0 +1,11 @@
+import { MovieType } from '../components/Movies/MovieTypes';

```

Comment:
What is your plan with this file? Currently, it is used by both the store and a component. Do you want to put every movie-related method here? It will be huge and won't conform with the SRP.
Filtering the movies could go into the slice. That is already imported into the components.
- getMovies is used by the store to fetch the data
- findMovies is used to filter the data within the store